packages/mdc-card/mdc-card.scss

```diff
@@ -23,15 +23,15 @@
 // postcss-bem-linter: define card
 
 .mdc-card {
-  @include mdc-card-fill-color(background);
+  @include mdc-card-fill-color(surface);
   @include mdc-card-corner-radius(2px);
   @include mdc-elevation(2);
   @include mdc-card-container-layout_;
 }
 
 .mdc-card--stroked {
   @include mdc-elevation(0);
-  @include mdc-card-stroke(#dbdbdb);
+  @include mdc-card-stroke(rgba(mdc-theme-prop-value(on-surface), .12));
```

Comment:
This will make the stroke translucent, use the Sass `mix()` function instead:
```
mix(mdc-theme-prop-value(on-surface), mdc-theme-prop-value(surface), 12%)
```
Note: the `mix()` function takes percentages so make sure to use 12%, not .12